x-pack/plugins/maps/public/components/layer_panel/settings_panel/settings_panel.js

```diff
@@ -28,14 +32,11 @@ export function SettingsPanel(props) {
     props.updateLabel(props.layerId, label);
   };
 
-  const onMinZoomChange = (event) => {
-    const zoom = parseInt(event.target.value, 10);
-    props.updateMinZoom(props.layerId, zoom);
-  };
-
-  const onMaxZoomChange = (event) => {
-    const zoom = parseInt(event.target.value, 10);
-    props.updateMaxZoom(props.layerId, zoom);
+  const onZoomChange = (value) => {
```

Comment:
I like how you did `[min, max]` in the change handler in size_range_selector.js better. Maybe do the same here to be consistent, and instead of min and max, how about lower and upper since they are not the min/max but the selected lower and upper values

Reply:
+1 , but keeping min/max naming for consistency with descriptors, action-creators